packages/frontend/src/components/common/Dashboard/DashboardHeader.tsx

```diff
@@ -180,6 +180,34 @@ const DashboardHeader = ({
         }),
     );
 
+    const handleDashboardRefreshUpdateEvent = useCallback(
+        (intervalMin?: number) => {
+            track({
+                name: EventName.DASHBOARD_AUTO_REFRESH_UPDATED,
+                properties: {
+                    useId: user.data?.userUuid,
+                    dashboardId: dashboardUuid,
+                    organizationId: organizationUuid,
+                    projectId: projectUuid,
+                    frequency: intervalMin ? `${intervalMin} minutes` : 'off',
+                },
```

Comment:
I believe that timestamp is already included in the track correct? I didn't see it getting passed anywhere